libraries/botbuilder-dialogs-adaptive-teams/tests/tests/actionTests.test.ts

```diff
@@ -0,0 +1,469 @@
+    };
+};
+
+const getBaseConversationReference = (): ConversationReference => {
+    return {
+        user: getTeamsUser(),
+        channelId: Channels.Msteams,
+        conversation: getPersonalConversation(),
+        bot: {
+            id: 'botId',
+            name: 'Bot',
+        },
+        serviceUrl: 'https://api.botframework.com',
+    };
+};
+
+const getPersonalConversationReference = (): ConversationReference => {
+    return {
+        ...getBaseConversationReference(),
+        conversation: getPersonalConversation(),
+    };
+};
+
+const getGroupConversationReference = (): ConversationReference => {
+    return {
+        ...getBaseConversationReference(),
+        conversation: getGroupConversation(),
+    };
+};
+
+const generateTeamMembers = (amount: number): Record<string, unknown>[] => {
+    const members = [];
+    const baseUser = getTeamsUser();
+    for (let i = 0; i < amount; i++) {
+        members.push({
+            id: `${baseUser.id}-${i}`,
+            name: `${baseUser.name}-${i}`,
+            objectId: `User-${i}-Object-Id`,
+            givenName: 'User',
+            surname: `Surname-${i}`,
+            email: `User.${i}@microsoft.com`,
+            userPrincipalName: `user${i}@microsoft.com`,
+            tenantId: 'tenant-id-1',
+        });
+    }
+
+    return members;
+};
+
+describe('Actions', function () {
+    ComponentRegistration.add(new AdaptiveTestComponentRegistration());
+    ComponentRegistration.add(new AdaptiveComponentRegistration());
+    ComponentRegistration.add(new TeamsComponentRegistration());
+
+    const resourceExplorer = new ResourceExplorer().addFolder(
+        path.join(__dirname, '../../tests/tests/actionTests'),
+        true,
+        false
+    );
+
+    /**
+     * Note: With mocha, `this.test?.title` refers to the test's name, so runTestScript
+     * is just calling a file with the same name as the test.
+     */
+    it('Action_GetMeetingParticipant', async function () {
+        const conversationReference = getPersonalConversationReference();
+        const participant = {
+            user: {
+                userPrincipalName: 'userPrincipalName-1',
+            },
+            meeting: {
+                role: 'Organizer',
+            },
+            conversation: conversationReference.conversation,
+        };
+
+        const fetchExpectation = nock('https://api.botframework.com')
+            .get('/v1/meetings/meeting-id-1/participants/participant-aad-id-1?tenantId=tenant-id-1')
+            .reply(200, participant);
+
+        const fetchExpectationCustomProperties = nock('https://api.botframework.com')
+            .get('/v1/meetings/customMeetingId/participants/customParticipantId?tenantId=customTenantId')
+            .reply(200, participant);
+
+        const adapter = getTeamsTestAdapter(conversationReference);
+
+        await TestUtils.runTestScript(resourceExplorer, this.test?.title, adapter);
+
+        ok(fetchExpectation.isDone());
```

Comment:
You can double check me here, but I think `isDone` includes an assertion internally.

Reply:
Couldn't F12 to the code, but ran a quick test and `isDone()` does not seem to `assert` anything. Test would pass when it shouldn't if I didn't wrap `isDone()` in `ok()`.